README.md

```diff
@@ -16,7 +16,7 @@ NOTE: Please, keep in mind that the access to the database is shared across ever
 ## Self-hosting
 
 The final setup consists of the following components:
-* PostgreSQL database (you can run it locally or in the cloud), which can hold the whole history of the blockchain (as of Decemeber 2021, mainnet takes 900GB of data in PostgreSQL storage, and testnet takes 375GB)
+* PostgreSQL database (you can run it locally or in the cloud), which can hold the whole history of the blockchain (as of January 2022, mainnet takes 2.5TB of data in PostgreSQL storage)
```

Comment:
Hmm, it takes 1.1TB for mainnet and 420GB for testnet on our side. We use PostgreSQL 11, and you?

Reply:
```suggestion
* PostgreSQL database (you can run it locally or in the cloud), which can hold the whole history of the blockchain (as of January 2022, mainnet takes 1.1TB of data in PostgreSQL storage, and testnet takes 420GB)
```